connectors/src/connectors/microsoft/temporal/file.ts

```diff
@@ -187,26 +187,23 @@ export async function syncOneFile({
 
   let documentSection: CoreAPIDataSourceDocumentSection | null = null;
   if (
-    mimeType ===
-    "application/vnd.openxmlformats-officedocument.presentationml.presentation"
+    [
+      "application/vnd.openxmlformats-officedocument.presentationml.presentation",
+      "application/pdf",
+    ].includes(mimeType)
   ) {
     const data = Buffer.from(downloadRes.data);
-    documentSection = await handlePptxFile(
-      data,
-      file.id,
-      localLogger,
-      maxDocumentLen
-    );
+    documentSection = await handleTextExtraction(data, localLogger, file);
```

Comment:
is handleTextExtraction handling pptx ? we don't need PPTX2Text anymore ?

Reply:
Yessir !